### DISCUSSION TITLE
Same .glb model, but shown differently in iOS and Android.

### DISCUSSION THREAD

**Discussion by streetcodec** [google/model-viewer#4551] [Q&A]
I have create a 3D model. A painting and it's frame. While it is working perfectly well in Android, there is strange error in iOS.
The iOS just doesn't show the back side of the frame, when viewing the frame upfront, but rotating the frame shows it. How to rectify it? I am adding examples to make the question more clear.
This is the Android Version, working perfectly fine : 
![WhatsApp Image 2023-11-01 at 12 22 23 PM](https://github.com/google/model-viewer/assets/92046906/89503455-3236-487f-a89d-79aeec5d1d98)
Now this is the iOS version. You can see space between outer frame and the painting.
![WhatsApp Image 2023-11-01 at 12 22 23 PM(1)](https://github.com/google/model-viewer/assets/92046906/1e3954ae-8e0c-4958-a9a7-8ef747663f90)
Now even stranger, if I rotate the object, the back side of the frame reveals itself.
![WhatsApp Image 2023-11-01 at 12 22 23 PM(2)](https://github.com/google/model-viewer/assets/92046906/cfc2f723-09d9-460c-9a02-b10ac93d0efa)
I have made the object inside blender, and used a single cube to make the frame. So how come this happens?

**Comment by elalish** [comment]
Best guess is you're using double-sided or back-sided materials. I'd recommend using only front-sided and then fix your triangle winding until the correct surfaces are shown. There are incompatibilities between glTF and USDZ in how double-sided is handled, so best to avoid relying on it.
